templates/taxi/driver_list.html

```diff
@@ -4,10 +4,13 @@
   <h1>Driver list</h1>
```

Comment:
According to the checklist, you should use 2 whitespaces for indentation in your `.html` files. Please adjust the indentation to improve the code readability.